cmd/apiserver/main.go

```diff
@@ -21,6 +21,7 @@ import (
 )
 
 const (
+	// PHD API
 	ENV_BACKEND_API_URL            = "BACKEND_API_URL"
```

Comment:
It's better to change the name of this env vars to actually be PHD related, current name is confusing

Reply:
Agree, but talking with Pascal we found out that this would mess with the deployment secrets, so we decided to just leave a comment for now